graylog2-web-interface/src/views/components/datatable/DataTable.jsx

```diff
@@ -100,20 +101,18 @@ const DataTable = ({ config, currentView, data, fields }: Props) => {
   return (
     <div className={styles.container}>
       <div className={styles.scrollContainer}>
-        <div className="messages-container">
-          <table className="table table-condensed messages">
-            <thead>
-              <Headers activeQuery={currentView.activeQuery}
-                       actualColumnPivotFields={actualColumnPivotFields}
-                       columnPivots={columnPivots}
-                       fields={fields}
-                       rollup={rollup}
-                       rowPivots={rowPivots}
-                       series={series} />
-            </thead>
-            {formattedRows}
-          </table>
-        </div>
+        <MessagesTable>
```

Comment:
I think this style definition can also be removed:
https://github.com/Graylog2/graylog2-server/blob/f0bb3dc08c0b1ec64445c6eea15f04351cdd50fd/graylog2-web-interface/src/theme/GlobalThemeStyles.jsx#L1279-L1282